zone/bot.cpp

```diff
@@ -3380,7 +3380,7 @@ void Bot::AI_Process()
 				TEST_COMBATANTS();
 				if (attack_timer.Check()) { // Process primary weapon attacks
 
-					Attack(tar, EQ::invslot::slotPrimary);
+					Mob::Attack(tar, EQ::invslot::slotPrimary);
```

Comment:
Bots inherit NPC which inherit Mob, is it really needed to prefix this? You should just have access to `Attack`

Reply:
NPC has a virtual override NPC::Attack, so when I was trying to access `Attack` it was defaulting to the NPC override.